Assets/Scripts/Entity/AttackLogic.cs

```diff
@@ -182,7 +182,11 @@ private void HitFinished()
                 // add damage boost from weapon if is player
                 if (_enemyController == null)
                     damageDealt += InventoryManager.Instance.DamageBoostFromEquipable;
-                Target.Hit(damageDealt, _enemyController);
+                Target.Hit(damageDealt, out int targetHealth, _enemyController);
+                if (targetHealth <= 0)
```

Comment:
I guess this shouldn't be called twice 🤔

Reply:
Oh dear, sorry 🤦‍♂